modules/xmpp/JingleSessionPC.js

```diff
@@ -1799,6 +1894,113 @@ export default class JingleSessionPC extends JingleSession {
     }
 
     /**
+     * Will put and execute on the queue a session modify task. Currently it
+     * only checks the senders attribute of the video content in order to figure
+     * out if the remote peer has video in the inactive state (stored locally
+     * in {@link _remoteVideoActive} - see field description for more info).
+     * @param {jQuery} jingleContents jQuery selector pointing to the jingle
+     * element of the session modify IQ.
+     * @see {@link _remoteVideoActive}
+     * @see {@link _localVideoActive}
+     */
+    modifyContents(jingleContents) {
```

Comment:
I will apply the simplification part, but not the work function part, because in all other 'modificationQueue' usages we're doing it this way and I don't want to make it different in one place now. There was another discussion about that with Brian when he was writing that initially, so I don't want to discuss it again.